### ISSUE TITLE
Missing CreolePluginModel

### ISSUE THREAD

**Issue by chrisspen** [pakal/cmsplugin-rst#1]
```
File "~/.env/local/lib/python2.7/site-packages/cmsplugin_rst/cms_plugins.py", line 14, in RstPlugin
    model = CreolePluginModel
NameError: name 'CreolePluginModel' is not defined
```

**Comment by ojii** [comment]
Hi,

I've not touched this plugin in two years and since I have no use for it anymore, I have no real interest in developing it any further. If you find this plugin useful and would like to take over the development, tell me your pypi username so I can hand over control to you.

**Comment by pakal** [comment]
Was fixed during the update of code to Djangocms 3.2
